package.json

```diff
@@ -70,7 +70,9 @@
     "audit:resolve": "SHELL=sh resolve-audit",
     "audit:check": "SHELL=sh check-audit",
     "dep:check": "npx ncu -e 2",
-    "dep:update": "npx ncu -u"
+    "dep:update": "npx ncu -u",
+    "snyk-protect": "snyk protect",
+    "prepare": "npm run snyk-protect"
```

Comment:
I don't like these npm commands, they don't conform to the pattern the other use. Perhaps they should be:
```
snyk:protect
snyk:prepare
```